src/QMCWaveFunctions/SPOSet.cpp

```diff
@@ -120,7 +134,8 @@ void SPOSet::evaluate_notranspose_spin(const ParticleSet& P,
                                        ValueMatrix_t& d2logdet,
                                        ValueMatrix_t& dspinlogdet)
 {
-  APP_ABORT("Need specialization of " + className + "::evaluate_notranspose_spin(P,iat,psi,dpsi,d2logdet, dspin_logdet) (vector quantities)\n");
+  APP_ABORT("Need specialization of " + className +
```

Comment:
runtime_error

Reply:
I went ahead and replaced all  the APP_ABORTS in the  file, not just this spin function

Reply:
Appreciated